intro-HPC/ch_account.tex

```diff
@@ -351,11 +351,10 @@ \subsection{Users of the \association}
 \fi % END IF ANTWERPEN
 
 \ifgent
-  You will now have to log in with CAS using your UGent account, and then be requested
-  to share your Mail and Affiliation with KU~Leuven (which actually means the VSC accountpage).
-  Select the ``Remember'' check-box and click ``Yes, continue''.
+  You will now have to log in with CAS using your UGent account, and then you
+  will be requested to share your information. Click ``Yes, continue''.
   \begin{center}
-  \includegraphics*[width=5.73in, height=4.00in, keepaspectratio=false]{ch2-browser-authenticate-gent}
+  \includegraphics*[width=\textwidth]{ch2-browser-authenticate-gent}
   \end{center}
```

Comment:
Add some extra information here:

* Your UGent login ID is either a login name of 7 characters (is that fixed?), or a (non-UGent) e-mail address if you are an external user.
* In case of problems with your UGent password, see https://password.ugent.be/

Reply:
It isn't fixed, mine has 8.

Reply:
Then use `8 or less`, that's probably right (or good enough in practice) ;)